### ISSUE TITLE
Cannot read properties of null (reading 'seconds')

### ISSUE THREAD

**Issue by martinocatur** [badeball/cypress-cucumber-preprocessor#903]
### Current behavior

<img width="470" alt="image" src="https://user-images.githubusercontent.com/23295412/207055314-f837a093-dc73-418f-b866-526418a41e59.png">

<img width="766" alt="Screen Shot 2022-12-12 at 20 23 15" src="https://user-images.githubusercontent.com/23295412/207056405-97900c40-8677-4004-ad00-62d723c4b63b.png">

### Desired behavior

No `properties of null (reading 'seconds')` error
### Test code to reproduce

<img width="433" alt="image" src="https://user-images.githubusercontent.com/23295412/207056724-ebf760ed-8fb5-4d21-965d-b6871702dc44.png">

Error appears in this block code after switch to `@badeball/cypress-cucumber-preprocessor` (same result when i tried on v14 and v15)

Previously, It works as expected using:
```
"cypress": "11.2.0",
"cypress-cucumber-preprocessor": "^4.0.0",
 ```

### Versions

* **Cypress version**: 11.2.0
* **Preprocessor version**: 15.0.0
* **Node version**: 19.0.1

### Checklist

- [x] I've read the [FAQ](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/faq.md).
- [x] I've read [Instructions for logging issues](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#instructions-for-logging-issues).
- [x] I'm not using `cypress-cucumber-preprocessor@4.3.1` (package name has changed and it is no longer the most recent version, see [#689](https://github.com/badeball/cypress-cucumber-preprocessor/issues/689)).

**Comment by martinocatur** [comment]
<img width="734" alt="image" src="https://user-images.githubusercontent.com/23295412/207075354-ae9e8182-f389-4157-8428-525b3ce1d61c.png">

not sure how can this happen

**Comment by badeball** [comment]
It's impossible to say given the provided information. You need to provide a complete and runnable example, so that I can observe the issue myself.

**Comment by fabio-grieco** [comment]
Hi, I got the same error. @badeball you can find a runnable example [here](https://github.com/fabio-grieco/dummy-cypress-cucumber-example). I'm using node v18.14.0 (npm v9.3.1).

**Comment by badeball** [comment]
https://docs.cypress.io/guides/references/error-messages#Cypress-detected-that-you-returned-a-promise-from-a-command-while-also-invoking-one-or-more-cy-commands-in-that-promise

tldr: dont use async/await.
